src/record/record.js

```diff
@@ -47,6 +52,27 @@ var Record = function( name, recordOptions, connection, options, client ) {
 EventEmitter( Record.prototype );
 
 /**
+ * Set a merge strategy to resolve any merge conflicts that may occur due
+ * to offline work or write conflicts. The function will be called with the 
+ * local record, the remote version/data and a callback to call once the merge has 
+ * completed or if an error occurs ( which leaves it in an inconsistent state until
+ * the next update merge attempt ).
+ *
+ * @param   {Function} mergeStrategy A Function that can resolve merge issues.
+ *
+ * @public
+ * @returns {void}
+ */
+Record.prototype.setMergeStrategy = function( mergeStrategy ) {
+	if( mergeStrategy instanceof Function ) {
```

Comment:
don't care too much personally, but there's been a few articles advising against checking instanceof primitives (e.g. when primitives are fiddled with in tests, when clients are distributed across browser windows etc... ) and rather use typeof... (similar thing with === undefined)